src/views/Team/index.view.tsx

```diff
@@ -1,12 +1,23 @@
-import * as React from "react"
+import React from "react"
 import "./index.scss"
 import NavBar from "../../components/NavBar/NavBar"
 import Footer from "../../components/Footer/Footer"
+import TeamGrid from "../../components/TeamCard/TeamCard"
 
 const TeamView: React.FC = () => (
   <div className='team-view'>
     <NavBar />
-    <p>Team Page Renders</p>
+    <div className='team-view__our-team'>OUR TEAM</div>
+    <div className='team-view__small-message'>
+      Meet our team of creators, thinkers,
+      <br />
+      strategists, and believers, collaborating
+      <br />
```

Comment:
In general, let's avoid using `<br />` tags. While the spacing is mainly is just the line-height, let's keep these a separate divs.